src/Actions/BuildModelDetails.php

```diff
@@ -89,11 +89,12 @@ public function __invoke(SplFileInfo $modelFile, bool $resolveAbstract = false):
     private function getModelDetails(SplFileInfo $modelFile, bool $resolveAbstract): ?array
     {
         $modelFileArg = $modelFile->getRelativePathname();
+        $modelFileArg = app()->getNamespace() . $modelFileArg;
```

Comment:
Need to give it the full name now so it can find things outside of `app/Models` so its now doing the equivalent of
```
php artisan model:show App/Models/User
```